packages/cmrjs/cmr-utils.js

```diff
@@ -504,6 +504,23 @@ function mergeURLs(original, updated = [], removed = []) {
   return [...unchangedOriginals, ...updatedWithMergedOriginals];
 }
 
+/**
+ * Updates CMR JSON file with stringified 'metadataObject'
+ *
+ * @param {Object} metadataObject - JSON Object to stringify
+ * @param {Object} cmrFile - cmr file object to write body to
+ */
+async function uploadUMMGJSONCMRFile(metadataObject, cmrFile) {
```

Comment:
This is a really good refactor. I think it needs a unit test, though.

Reply:
00af1f71826fa072b0ce4eebd9e0d5b5de97c8b9